packages/@react-spectrum/dialog/docs/Dialog.mdx

```diff
@@ -0,0 +1,329 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/dialog';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ActionButton, Button} from '@react-spectrum/button';
+import {Content, Footer, Header} from '@react-spectrum/view';
+import {Checkbox} from '@react-spectrum/checkbox';
+import {Dialog, DialogTrigger} from '@react-spectrum/dialog';
+import {Divider} from '@react-spectrum/divider';
+import {Form} from '@react-spectrum/form';
+import {Heading, Text} from '@react-spectrum/typography';
+import {Image} from '@react-spectrum/image';
+import {TextField} from '@react-spectrum/textfield';
+```
+
+# Dialog
+
+<p>{docs.exports.Dialog.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Dialog']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/dialog/'}
+  ]} />
+
+## Example
+
+```tsx example
+<DialogTrigger>
+  <ActionButton>Trigger</ActionButton>
+  {(close) => (
+    <Dialog>
+      <Header><Heading>The Heading</Heading></Header>
+      <Divider />
+      <Content>
+        <Text>
+          This is a dialog.
+        </Text>
+      </Content>
+      <Footer>
+        <Button variant="secondary" onPress={close}>Cancel</Button>
+        <Button variant="cta" onPress={close}>Confirm</Button>
+      </Footer>
+    </Dialog>
+  )}
+</DialogTrigger>
+```
+
+## Content
+
+The layout of a Dialog is made up of the following elements and sections, mapped to the corresponding default slot names:
+
+| Section/Element 	|     Slot      |
+|-------------------|--------------:|
+| Container       	| container   	|
+| Cover Image     	| hero        	|
+| Header          	| header      	|
+| Heading         	| heading     	|
+| Type Icon       	| typeIcon    	|
+| Divider         	| divider     	|
+| Body            	| content     	|
+| Footer          	| footer      	|
+| Close Button    	| closeButton 	|
+
+![Image of Dialog Layout](./images/dialog.png)
+INSERT PICTURE HERE, MAYBE LINK TO SPECTRUM (Contributions is a corp site)
```

Comment:
Readding comment:
Still trying to figure out how to format this table so it isn't so small and so it can be centered. Also trying to figure out how to get the image to render.

Open to opinions on if the table or the picture is a good idea. I would link to the contributions site but its on the corp network.